### PR TITLE
dotCMS/core#19987 Fixing bug

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
     (hide)="removeFieldsWithoutId()"
     [header]="currentFieldType?.label"
 >
-    <p-tabView *ngIf="displayDialog" (onChange)="handleTabChange($event.index)">
+    <p-tabView (onChange)="handleTabChange($event.index)">
```
Comment (fmontes) [inline src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html:41]: Why this fix it?

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.spec.ts [hunk 2]
```diff
@@ -225,6 +225,10 @@ describe('ContentTypeFieldsDropZoneComponent', () => {
         })
     );
 
+    fit('should have propertiesForm', () => {
```
Comment (fmontes) [inline src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.spec.ts:228]: Remove fit.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by alfredo-dotcms — APPROVED** [review]

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html [hunk 3]
```diff
@@ -38,7 +38,7 @@
     (hide)="removeFieldsWithoutId()"
     [header]="currentFieldType?.label"
 >
-    <p-tabView *ngIf="displayDialog" (onChange)="handleTabChange($event.index)">
+    <p-tabView [(activeIndex)]="activeTab" (onChange)="handleTabChange($event.index)">
```
Comment (fmontes) [inline src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html:41]: needs testing.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html (hunk 3 above)
Comment (freddyucv) [reply src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html:41]: done

File: src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html (hunk 1 above)
Comment (freddyucv) [reply src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html]: done

**Review by fmontes — APPROVED** [review]

**Pull request merged**